### PR TITLE
sql-statement: update `admin` related statements

### PR DESCRIPTION
**Pull request by zimulala**
### What is changed, added or deleted? (Required)
Update `admin` related statements

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s): pingcap/docs-cn#3175

### CODE REVIEW

File: sql-statements/sql-statement-admin.md [hunk 1]
```diff
@@ -64,6 +68,102 @@ ADMIN CHECK TABLE tbl_name [, tbl_name] ...;
...
+
+The above statement can generate the binding of SQL Plan from the `SELECT` statement that occurs more than once.
+
+{{< copyable "sql" >}}
+
+```sql
+ADMIN EVOLVE bindings;
+```
+
+After the automatic binding function is turned on, the evolution of SQL Plan binding information is triggered every ` bind-info-leave` (default value is `3s`). The above statement is used to actively trigger this evolution.
```
Comment (toutdesuite) [inline sql-statements/sql-statement-admin.md:155]:
```suggestion
After the automatic binding feature is enabled, the evolution of SQL Plan binding information is triggered every ` bind-info-leave` (the default value is `3s`). The above statement is used to proactively trigger this evolution.
```

File: sql-statements/sql-statement-admin.md [hunk 2]
```diff
@@ -74,6 +174,22 @@ ADMIN REPAIR TABLE tbl_name CREATE TABLE STATEMENT;
...
+ADMIN SHOW SLOW RECENT N;
+```
+
+{{< copyable "sql" >}}
+
+```sql
+ADMIN SHOW SLOW TOP [INTERNAL | ALL] N;
+```
+
+For detailed usage, refer to [admin show slow statement] (/identify-slow-queues.md#admin-show-slow-command)
```
Comment (toutdesuite) [inline sql-statements/sql-statement-admin.md:191]:
```suggestion
For details, refer to [admin show slow statement](/identify-slow-queues.md#admin-show-slow-command)
```

File: sql-statements/sql-statement-admin.md [hunk 3]
```diff
@@ -74,6 +174,22 @@ ADMIN REPAIR TABLE tbl_name CREATE TABLE STATEMENT;
...
+ADMIN SHOW SLOW RECENT N;
+```
+
+{{< copyable "sql" >}}
+
+```sql
+ADMIN SHOW SLOW TOP [INTERNAL | ALL] N;
+```
+
+For details, refer to [admin show slow statement](/identify-slow-queues.md#admin-show-slow-command)
```
Comment (yikeke) [inline sql-statements/sql-statement-admin.md:191]:
```suggestion
For details, refer to [admin show slow statement](/identify-slow-queries.md#admin-show-slow-command)
```

**Comment by yikeke** [comment]
I fixed the heading levels in https://github.com/pingcap/docs-cn/pull/3712/.

File: sql-statements/sql-statement-admin.md [hunk 4]
```diff
@@ -64,6 +76,104 @@ ADMIN CHECK TABLE tbl_name [, tbl_name] ...;
...
+
+### `admin checksum`
+
+{{< copyable "sql" >}}
+
+```sql
+ADMIN CHECKSUM TABLE tbl_name [, tbl_name] ...;
+```
+
+The above statement is used to get the 64-bit checksum value of `tbl_name'. This value is obtained by calculating CRC64 of all key value pairs (including row data and index data) in the table.
```
Comment (yikeke) [inline sql-statements/sql-statement-admin.md:103]:
```suggestion
The above statement is used to get the 64-bit checksum value of `tbl_name`. This value is obtained by calculating CRC64 of all key-value pairs (including row data and index data) in the table.
```

**Pull request merged**